apps/kyb-app/entrypoint.sh

```
@@ -0,0 +1,46 @@
+#!/usr/bin/env sh
+
+if [[ -z "$VITE_DOMAIN" ]]
```

Shellcheck

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)